config/Dockerfiles/tests.d/aws/05_aws_ec2_vpc_endpoint

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -xe
+
+# Verify the aws ec2 vpc subnet provisioning
+# distros.exclude: centos7 fedora27 fedora28
+# providers.include: aws
+# providers.exclude: none
+
+DISTRO=${1}
+PROVIDER=${2}
+
+TARGET="aws-ec2-vpc-endpoint"
+
+function clean_up {
+    set +e
+    linchpin -w . -v destroy "${TARGET}"
+}
+trap clean_up EXIT SIGHUP SIGINT SIGTERM
+
+pushd docs/source/examples/workspaces/${PROVIDER}
+
+echo "add default profile in config"
+mkdir -p ~/.aws/
+printf "[default]\nregion = us-east-1" > ~/.aws/config
```

Comment:
Just curious why we have to do this?

Reply:
@p3ck : I was getting the following [error](https://stackoverflow.com/questions/34134879/aws-the-config-profile-myname-could-not-be-found )
aws_ec2_vpc_endpoint was looking for a default profile in the config . I had to create that to resolve the error.

Reply:
We don't need this for the other aws providers?  Does this need to be added to this providers documentation in linchpin?

Reply:
No we donot need this for other modules . However, In my local testing I tried removing the file and it was working fine. I believe it is specific to this module in our container environment.